### ISSUE TITLE
Cat halth falls below certain level

### ISSUE THREAD

**Issue by Nicka244** [runelite/runelite#3337]
I was hoping someone could implement a notification for cat health when they are farming spices. This would help people who lose their cats while trying to farm spices for stews and boosts.

**Comment by patriqdesigns** [comment]
If no one would mind I can implement this :)

**Comment by StephanBijzitter** [comment]
@patriqdesigns the overhead chat lines from a car represent information as well. It's far from perfect, but everything combined could make it pretty accurate.

>Text above the kitten can also indicate its status; a long "Meeeooowww!" will mean it is very hungry, for example, while the same drawn out meow but with a "..." indicates it is feeling lonely.

http://oldschoolrunescape.wikia.com/wiki/Cat

**Comment by patriqdesigns** [comment]
Yeah I have been raising a cat since yesterday to see what I could possibly
use.
I still have to check how can I check for overhead chat lines, but I am
unaware of how would I map it to a number that states how hungry the cat is.
Maybe the notification could just show up when he is really hungry or
lonely?

On Mon, May 28, 2018 at 3:51 PM, Stephan Bijzitter <notifications@github.com
